lib/routes/home/widgets/payments_list/payment_item_avatar.dart

```diff
@@ -34,7 +40,35 @@ class PaymentItemAvatar extends StatelessWidget {
     final details = paymentItem.details.data;
     final isKeySend = (details is LnPaymentDetails) ? details.keysend : false;
     final isClosedChannelPayment = paymentItem.paymentType == PaymentType.ClosedChannel;
+    bool hasMetadataDescription = false;
+    if (details is LnPaymentDetails && details.lnurlMetadata != null) {
+      final metadataMap = {
+        for (var v in json.decode(details.lnurlMetadata!)) v[0] as String: v[1],
```

Comment:
This parsing repeats itself for some properties we extract from the medata.
I think a better approach is to parse the metadata once in the constructor and save it as a Map field.
Also if json parsing failed we should fallback to empty map.
When properties are needed (image, description) we should just access the parsed map.